node-fs-writefile/cp.js

```diff
@@ -0,0 +1,12 @@
+import fs from 'node:fs/promises';
+
+const filePath = process.argv[2];
+const newFileName = process.argv[3];
+const content = fs.readFile(filePath, 'utf-8');
```

Comment:
For a copy, you don't want to convert the contents to 'utf-8' so just leave off the second parameter.